src/components/map/layers/NodeLayer.tsx

```diff
@@ -43,31 +55,40 @@ export default class NodeLayer extends Component<MarkerLayerProps> {
             type = node.type;
         }
 
+        if (node.stop && node.stop.hastusId && this.props.mapStore!.zoom >= HASTUS_MIN_ZOOM) {
+            children.push(
+                this.getLabelHtml(node.stop.hastusId),
+            );
+        }
+
         let html;
         switch (type) {
         case NodeType.STOP: {
-            html = this.getMarkerHtml(isSelected ? s.stopMarkerHighlight : s.stopMarker);
+            html = this.getMarkerHtml(isSelected ? s.stopMarkerHighlight : s.stopMarker, children);
             break;
         }
         case NodeType.CROSSROAD: {
-            html = this.getMarkerHtml(isSelected ? s.crossroadMarkerHighlight : s.crossroadMarker);
+            html = this.getMarkerHtml(
+                isSelected ? s.crossroadMarkerHighlight : s.crossroadMarker, children);
             break;
         }
         case NodeType.MUNICIPALITY_BORDER: {
-            html = this.getMarkerHtml(isSelected ?
-                s.municipalityMarkerHighlight : s.municipalityMarker);
+            html = this.getMarkerHtml(
+                isSelected ? s.municipalityMarkerHighlight : s.municipalityMarker, children);
             break;
         }
         case NodeType.DISABLED: {
-            html = this.getMarkerHtml(isSelected ? s.disabledMarkerHighlight : s.disabledMarker);
+            html = this.getMarkerHtml(
+                isSelected ? s.disabledMarkerHighlight : s.disabledMarker, children);
             break;
         }
         case NodeType.TIME_ALIGNMENT: {
-            html = this.getMarkerHtml(s.timeAlignmentMarker);
+            html = this.getMarkerHtml(s.timeAlignmentMarker, children);
             break;
         }
         default: {
-            html = this.getMarkerHtml(isSelected ? s.unknownMarkerHighlight : s.unknownMarker);
+            html = this.getMarkerHtml(
+                isSelected ? s.unknownMarkerHighlight : s.unknownMarker, children);
             break;
         }
```

Comment:
I think this whole method getIcon() would need rewriting as now it is starting to look complex

according to https://stackoverflow.com/a/34776015 we could aim for a template of:

```
<div>
<div>marker</div>
<div>label</div>
</div>
```

```
getIconHtml() {
    return `
       this.getMarkerHtml()
       this.getLabelHtml()
`
}
```

and have that switch clause in getMarkerHtml & getLabelHtml